syntax = "proto3";

service SetUserApns {
    rpc SetUserApns(SetUserApnsRequest) returns (SetUserApnsResponse);
}

message SetUserApnsRequest {
    string sid = 1;
    string apn = 2;
}

message SetUserApnsResponse {
    int32 status = 1;
    string msg = 2;
}